src/routing/bootstrap.rs

```diff
@@ -21,18 +21,19 @@ use crate::{
 };
 use bytes::Bytes;
 use futures::future;
+use resource_proof::ResourceProof;
 use std::{collections::VecDeque, mem, net::SocketAddr};
 use tokio::sync::mpsc;
 use xor_name::Prefix;
 
 const BACKLOG_CAPACITY: usize = 100;
 
-/// Bootstrap into the network as an infant node.
+/// Bootstrap into the network as an adult node.
```

Comment:
Maybe "Bootstrap into the network for the first time" or "...as new node" or something like that. Again, it's not important that we are adult, but that we are not being relocated.